src/collective/fingerpointing/subscribers/registry_logger.py

```diff
@@ -23,6 +23,6 @@ def registry_logger(event):
 
         if IRecordModifiedEvent.providedBy(event):
             action = 'modify'
-            extras = 'object={0} value={1}'.format(event.record, event.record.value)
+            extras = u'object={0} value={1}'.format(event.record, event.record.value)
```

Comment:
not sure if we don't need to add safe_unicode(event.record.value) here to be completely safe.

if it can happen that utf-8 encoded texts are passed to the registry than this will lead to a unicodedecoderror:

```
>>> st=u'Ümläut'.encode('utf-8')
>>> u'Foo {}'.format(st)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 0: ordinal not in range(128)
```